docs/OBSIDIAN-SETUP.md

```
@@ -1,183 +1,117 @@
-# Frontmatter template (Property Template)
-title: ""
-description: ""
-pubDate: {{date:YYYY-MM-DD}}
-tags: []
-draft: true
-
-# File settings
-Blog Folder: src/content/writing
-Use Date Folders: true
-Date Folder Format: YYYY/MM
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)